public/app/features/users/UsersTable.tsx

```diff
@@ -31,16 +31,20 @@ const UsersTable: FC<Props> = props => {
         {users.map((user, index) => {
           return (
             <tr key={`${user.userId}-${index}`}>
-              <td className="width-4 text-center">
+              <td className="width-2 text-center">
                 <img className="filter-table__avatar" src={user.avatarUrl} />
               </td>
-              <td>{user.login}</td>
+              <td className="max-width-6">
+                <span className="ellipsis">{user.login}</span>
```

Comment:
Great! Can you also add `title` attributes to those that have ellipsis, so that the user can see what's the full content when hovering over?